examples/funcalign/requirements.txt

```diff
@@ -1,2 +1,3 @@
 matplotlib
+nilearn
```

Comment:
Could you explain why this dependency just appeared, or why it was not here before?